torch/nn/modules/distance.py

```diff
@@ -1,6 +1,29 @@
+from .module import Module
+from .. import functional as F
+
+
+class PairwiseDistance(Module):
+    """
+    Computes the batchwise pairwise distance between vectors v1,v2:
+
+        .. math ::
+            \Vert x \Vert _p := \left( \sum_{i=1}^n  \vert x_i \vert ^ p \right) ^ {1/p}
+
+        Inputs: x
```

Comment:
We're using a different convention for inputs and outputs (`Arguments` or `Args` and `Returns`). Please make it consistent with documentation of other modules.